### PR TITLE
OpenJDK 11

### PR DESCRIPTION
**Pull request by m4rc1no5**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,8 +4,14 @@ MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 # User root user to install software
 USER root
 
-# Install necessary packages
-RUN yum -y install java-1.8.0-openjdk-devel && yum clean all
+# Install OpenJDK 11
+WORKDIR /opt
+RUN yum install wget -y \
+	&& wget https://download.java.net/java/ga/jdk11/openjdk-11_linux-x64_bin.tar.gz \
```
Comment (goldmann) [inline Dockerfile]: Why not using OpenJDK provided in CentOS?

Comment (m4rc1no5) [reply Dockerfile]: I don't find RPM package with OpenJDK 11 in Centos repositories.

Comment (goldmann) [reply Dockerfile]: Take a look at the `java-11-openjdk-devel` package. It's new in CentOS 7.6.

Comment (m4rc1no5) [reply Dockerfile]: You're right, thank you :)

**Review by goldmann — APPROVED** [review]

**Comment by goldmann** [comment]
Squashed and merged, thanks!

**Pull request merged**